src/main/java/org/craftercms/studio/model/rest/ApiResponse.java

```diff
@@ -171,6 +171,10 @@ public class ApiResponse {
     public static final ApiResponse AWS_UNREACHABLE = new ApiResponse(52000, "AWS is unreachable",
             "Advise the user that AWS engine is not reachable", StringUtils.EMPTY);
 
+    // 53000 - 54000
+    public static final ApiResponse LOGGER_NOT_FOUND = new ApiResponse(53000, "The logger was not found",
```

Comment:
Please also add to the studio-api-2.yaml, see this: https://app.swaggerhub.com/apis/craftercms/studio/4.0.1.0#/ApiResponse